docker/incremental_load.sh.tpl

```diff
@@ -18,14 +18,24 @@ set -eu
 
 # This is a generated file that loads all docker layers built by "docker_build".
 
+# Check we are using GNU tar as this script relies on some impl specifics
+# e.g. on a Mac the default is a BSD variant which will bork
+tar --version | grep 'GNU' >/dev/null 2>&1
+if [ ! $? -eq 0 ]; then
+  echo "Error: GNU tar needs to be installed."
+  echo "If you are on a Mac, install Homebrew then..."
+  echo "  brew install gnu-tar --with-default-names"
+  exit 1
+fi
```

Comment:
That's the one.

100% with you that it should go.  However I saw #72 and figured that was the desired mid/long term approach.  If that ticket happens then this will become mute.  Will create an issue in a minute anyway so it what we've discussed here doesn't get lost to the sands of time.